pythonparser/lexer.py

```diff
@@ -418,28 +419,41 @@ def _string_literal(self, options, begin_span, data, data_span, end_span):
                           "strend"))
 
     def _replace_escape(self, range, mode, value):
-        is_raw     = ("r" in mode)
```

Comment:
Done. Now is_unicode and is_byte are the inverse of each other so I've left out is_byte.